spring-data-jdbc/src/test/java/org/springframework/data/jdbc/core/convert/SqlGeneratorUnitTests.java

```diff
@@ -220,11 +220,11 @@ public void findAllSortedBySingleField() {
 				"ORDER BY x_name ASC");
 	}
 
-	@Test // DATAJDBC-101
+	@Test // DATAJDBC-101, DATAJDBC-584
```

Comment:
I'd prefer to have a separate test case for this. It makes it easier to understand what is going wrong in case of a test failure.